.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: LAPIS-SILO
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  dockerImage:
+    name: 'Build Docker image'
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' }}
```

Comment:
Why should the workflow be executed on both push and pull_request (lines 4-5) but then, the job only runs on push?

Reply:
Because this will contain additional jobs for the tests etc. (at least that's what I had in mind.